### PR TITLE
change deprecated `@elseif` to `@else if`

### PR DESCRIPTION
**Pull request by grafmouse**

### CODE REVIEW

File: test/css/main.css [hunk 1]
```diff
@@ -777,22 +222,22 @@ html {
 
 .rem {
   font-size: 20.8px;
-  font-size: 1.3rem;
-  padding: 20px;
+  font-size: 1.3 * 1rem;
```
Comment (drublic) [inline test/css/main.css]: This looks like something is wrong here.

File: test/css/main.css [hunk 2]
```diff
@@ -738,13 +180,16 @@ html {
...
+
 .placeholder::-moz-placeholder {
   color: #bada55; }
+
 .placeholder:-ms-input-placeholder {
   color: #bada55; }
 
 .opacity {
   opacity: 0.4;
+  -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=40)";
```
Comment (drublic) [inline test/css/main.css]: I think we don’t need to support old IE.

File: test/css/main.css [hunk 3]
```diff
@@ -777,22 +222,22 @@ html {
...
-  font-size: 1.3rem;
-  padding: 20px;
+  font-size: 1.3 * 1rem;
+  padding: 20 * 1px;
   padding: 1.25rem; }
 
 .rem-2 {
-  padding: 16px 0 48px;
-  padding: 1rem 0 3rem; }
+  padding: 16 * 1px 0 48px;
```
Comment (drublic) [inline test/css/main.css]: Same as above. also see below.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,9 @@
 
 ### HEAD
 
+### 0.13.0 - 31.01.2019
```
Comment (drublic) [inline CHANGELOG.md]: I think i need to do another version here after merging. so you can skip this line for now.

**Review by drublic — CHANGES_REQUESTED** [review]
Thanks for the nice contribution. Please update the things I outlined and we’re good to merge :)

**Comment by towry** [comment]
https://github.com/drublic/Sass-Mixins/issues/50

**Comment by drublic** [comment]
Merged.

**Pull request closed without merging**